app/javascript/slices/greetingSlice.js

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const greetingSlice = createSlice({
+  name: 'greeting',
+  initialState: {
+    value: '',
+  },
+  reducers: {
+    setGreeting: (state, action) => {
+      state.value = action.payload;
+    },
+  },
+});
+
+export const { setGreeting } = greetingSlice.actions;
```

Comment:
- [ ] As I mentioned above, please adhere to the project requirements, instead of fetching data from `API` using the `useEffect` hook, you should implement the `redux` thunk as instructed in the project requirement, with the help of which you can perform side-effects. Kindly implement the projects according to project requirements.

![image](https://github.com/Omwomo/hello-rails-react/assets/66476329/d2901dd3-7653-48fc-93ff-156c9630d38d)